src/libraries/Native/Unix/System.Native/pal_networkchange.c

```diff
@@ -111,9 +108,6 @@ void SystemNative_ReadEvents(int32_t sock, NetworkChangeEvent onNetworkChange)
             case RTM_NEWLINK:
                 onNetworkChange(sock, ReadNewLinkMessage(hdr));
                 break;
-            case RTM_DELLINK:
-                onNetworkChange(sock, LinkRemoved);
-                break;
```

Comment:
Why is handling for RTM_DELLINK no longer needed?

Reply:
Similar `LinkRemoved` was not handled on managed side, so I removed it here also.